crates/re_data_ui/src/item_ui.rs

```diff
@@ -248,8 +219,13 @@ fn cursor_interact_with_selectable(
 
     if response.hovered() {
         selection_state.set_hovered(std::iter::once(item));
+    } else if response.clicked() {
+        if response.ctx.input(|i| i.modifiers.command) {
+            selection_state.toggle_selection(selection_state.hovered().to_vec());
+        } else {
+            selection_state.set_multi_selection(selection_state.hovered().clone().into_iter());
```

Comment:
Inlining `select_hovered_on_click` here will make it more difficult to change our selection behavior (e.g. change from the command key to the shift key)